docs/explanations/frame.rst

```diff
@@ -0,0 +1,103 @@
+.. _frame-theory:
+
+===========================
+Frame theory and dual space
+===========================
+
+.. seealso::
+
+   This content is adapted from the work of L. E. Fischer, T. Dao, I. Tavernelli,
+   and F. Tacchino, `Dual-frame optimization for informationally complete
+   quantum measurements <https://journals.aps.org/pra/abstract/10.1103/PhysRevA.109.062415>`_, Phys. Rev. A 109 (2024).
+
+As a reminder, given an IC-POVM :math:`\mathbf{M} = \{M_k\}_{k \in \{1, \dots, n \}}` and an
+observable :math:`\mathcal{O}`, there exist :math:`\omega_k \in \mathbb{R}` such that
+
+.. math::
+   :label: observable_povm_decomposition
+
+   \mathcal{O}= \sum_{k=1}^{n} \omega_k M_k .
+
+We will now outline a formal approach to obtain the coefficients
+:math:`\omega_k` in Eq. :eq:`observable_povm_decomposition` for
+a given observable :math:`O`. First, we note that the minimal number of
+linearly independent POVM elements for an IC-POVM is :math:`n = d^2`. We
+call such POVMs *minimally informationally complete*.
+In that case, the coefficients :math:`\omega_k` are unique. However, for
+POVMs with :math:`n > d^2`, such as those that arise from IC PM-simulable
+POVMs, the decomposition in
+Eq. :eq:`observable_povm_decomposition` is not unique. This
+redundancy is described by frame theory, as outlined in
+Ref. [#innocenti2023shadow]_.
+
+The set of POVM operators
+:math:`\mathbf{M} = \{M_k\}_{k \in \{1, \dots, n \}}` forms a frame for the
+space of Hermitian operators if and only if it is IC. For any frame,
+there exists at least one dual frame
+:math:`\mathbf{D} = \{D_k\}_{k \in \{1, \dots, n \}}`, such that
+
+.. math::
+   :label: eqn:definition_duals  
+   
+   O = \sum_{k=1}^n \mathrm{Tr}[OD_k] M_k
```

Comment:
Above we use `\mathcal{O}` for the observable. Is there any reason not to do so here (and below)?